tests/test_other.py

```diff
@@ -2296,6 +2296,30 @@ def clean(txt):
     out3, err3 = Popen([PYTHON, FILE_PACKAGER, 'test.data', '--preload', 'data2.txt', 'data2.txt@waka.txt'], stdout=PIPE, stderr=PIPE).communicate()
     out3 = clean(out3)
     assert out != out3
+    # verify '--separate-metadata' option produces separate metadata file
+    os.chdir('..')
+    Popen([PYTHON, FILE_PACKAGER, 'test.data', '--preload', 'data1.txt', '--preload', 'subdir/data2.txt', '--js-output=immutable.js', '--separate-metadata']).communicate()
+    assert os.path.isfile('immutable.js.metadata')
+    # verify js output file is immutable when metadata is separated
+    shutil.copy2('immutable.js', 'immutable.js.copy')
```

Comment:
better to make this a `.move`, both for speed, and to ensure it doesn't exist before the next line (to check it actually emits it)